### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by tYuriy**

### CODE REVIEW

**Comment by tYuriy** [comment]
[DEMO LINK](https://tYuriy.github.io/react_static-list-of-todos/)

File: src/components/TodoList/TodoList.tsx [hunk 1]
```diff
@@ -1,4 +1,13 @@
...
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList:React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(todo => <TodoInfo todo={todo} />)}
+  </section>
```
Comment (vlad-demchuk) [inline src/components/TodoList/TodoList.tsx:12]: it's not sections and articles. it's just a list, so use `li, ull` here

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]

File: src/components/TodoInfo/TodoInfo.tsx [hunk 2]
```diff
@@ -1,4 +1,29 @@
...
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title, completed, user,
+  } = todo;
+
+  return (
+    <>
```
Comment (vlad-demchuk) [inline src/components/TodoInfo/TodoInfo.tsx:15]: you don't need react fragment here

File: src/components/UserInfo/UserInfo.tsx [hunk 3]
```diff
@@ -1,4 +1,17 @@
...
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const { email, name } = user;
+
+  return (
+    <>
```
Comment (vlad-demchuk) [inline src/components/UserInfo/UserInfo.tsx:11]: and this one

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]

File: src/components/TodoList/TodoList.tsx [hunk 4]
```diff
@@ -7,7 +8,17 @@ type Props = {
...
+    {todos.map(todo => (
+      <li
+        key={todo.id}
+        className={classNames(
+          'TodoInfo',
+          { 'TodoInfo--completed': todo.completed },
+        )}
+      >
+        <TodoInfo todo={todo} />
+      </li>
```
Comment (nazarmatsevych) [inline src/components/TodoList/TodoList.tsx:21]: move this part(from 13 to 21 lines) to TodoInfo component

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]

**Review by nazarmatsevych — APPROVED** [review]

**Pull request closed without merging**